code/__DEFINES/mobs.dm

```diff
@@ -440,6 +440,13 @@
 #define MAX_REVIVE_FIRE_DAMAGE 180
 #define MAX_REVIVE_BRUTE_DAMAGE 180
 
+#define DEFAULT_BRUTE_EXAMINE_TEXT "bruising"
+#define DEFAULT_BURN_EXAMINE_TEXT "burns"
+#define DEFAULT_CLONE_EXAMINE_TEXT "cellular damage"
+
+#define ROBOTIC_BRUTE_EXAMINE_TEXT "denting"
+#define ROBOTIC_BURN_EXAMINE_TEXT "charrings"
```

Comment:
```suggestion
#define ROBOTIC_BURN_EXAMINE_TEXT "charring"
```

Reply:
It's done to be consistent with burns, 

``He has moderate charrings.`` vs ``He has moderate charring.``